db/migrate/20200703152351_create_prosecution_case_hearings.rb

```diff
@@ -0,0 +1,20 @@
+class CreateProsecutionCaseHearings < ActiveRecord::Migration[6.0]
+  def change
+    create_table :prosecution_case_hearings, id: :uuid do |t|
+      t.references :prosecution_case, type: :uuid, null: false, foreign_key: true
+      t.references :hearing, type: :uuid, null: false, foreign_key: true
+
+      t.timestamps
+    end
+    reversible do |dir|
+      dir.up do
+        execute <<-SQL
```

Comment:
**not a blocker** - if you were not aware u can use squiggly heredocs to allow for visual indentation without indentation

```
page = <<-HTML
  Heredocs are cool & useful
HTML

# "  Heredocs are cool & useful\n"

page = <<~HTML
  Heredocs are cool & useful
HTML

# "Heredocs are cool & useful\n"
```

Reply:
Thank you, I wasn't very familiar with the squiqqly version!